### ISSUE TITLE
[Accessibility] No focus visual render on the buttons of the Explorer toolbox

### ISSUE THREAD

**Issue by veler** [Microsoft/pxt#2499]
I realize that we don't have any visual render when the Add/Trash buttons of the Explorer toolbox have the focus.
At the moment where I write this issue, I'm writing some tests scenarios where I explain that after a Tab key, the Add Custom Blocks button must have the focus. It has the focus, but no visual render and even if it is a detail I feel it would be relevant to fix it.

**Comment by abchatra** [comment]
closed, filed by us.
